server/build.gradle

```diff
@@ -13,7 +13,7 @@ apply plugin: 'elasticsearch.internal-cluster-test'
 
 publishing {
   publications {
-    nebula {
+    elastic(MavenPublication) {
```

Comment:
Why is it necessary to specify the type here? Isn't that only when _creating_ a new container object? In this context we're configuring an existing publication, no?

Reply:
yes. that was an oversight of an earlier implementation where I created the publication in an afterEvaluate. But I moved it to be done immediately to avoid exactly this. yeah we should only configure existing publications here.